python/dgl/data/ppi.py

```diff
@@ -0,0 +1,60 @@
+"""PPI Dataset.
+(zhang hao): Used for inductive learning.
+"""
+
+import numpy as np
+from .utils import download, extract_archive, get_download_dir, _get_dgl_url
+from sklearn.preprocessing import StandardScaler
+import networkx as nx
+from networkx.readwrite import json_graph
+import json
+
+_url = 'dataset/ppi.zip'
+
+class ppi_dataset(object):
```

Comment:
`PPIDataset`

Reply:
Also more docstring. Explain what is each sample looks like.

Reply:
Also, try to make it compatible with torch's Dataset interface. Usually, you need to at least implement `__getitem__` and `__len__`. You could have a look at the [MiniGCDataset](https://docs.dgl.ai/api/python/data.html#mini-graph-classification-dataset).